lib/directory.js

```diff
@@ -94,15 +97,15 @@ exports.handler = function (route, options) {
 
         // Generate response
 
-        var resource = request.path;
-        var hasTrailingSlash = (resource[resource.length - 1] === '/');
-        var fileOptions = { lookupCompressed: settings.lookupCompressed, etagMethod: settings.etagMethod };
+        const resource = request.path;
+        const hasTrailingSlash = (resource[resource.length - 1] === '/');
```

Comment:
Maybe we can use `resource.endsWith('/')` for that, @cjihrig @kpdecker any clue about performance on this ?